python/mxnet/base.py

```diff
@@ -734,3 +734,8 @@ def write_all_str(module_file, module_all_list):
 
 ctypes.pythonapi.PyCapsule_New.restype = ctypes.py_object
 ctypes.pythonapi.PyCapsule_GetPointer.restype = ctypes.c_void_p
+
+from .runtime import Features
+if Features().is_enabled("TVM_OP"):
```

Comment:
Seems TVM is supported by python only because of this code. Shall we load libtvmop automatically along with loading libmxnet? Then we don't do extra change with front-end language.

Reply:
It should be doable. while might need non-trivial surgery to both. cc @tqchen if you have insight.